dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -160,7 +161,7 @@ public class Contentlet implements Serializable, Permissionable, Categorizable,
   public static final String RELATIONSHIP_KEY = "__##relationships##__";
 
   private transient ContentType contentType;
-  protected Map<String, Object> map = new ContentletHashMap();
```

Comment:
why are you removing this?

Reply:
It was not removed. I only removed the unnecessary initialization. The `map` variable is still there.